plugins/certs/help-functions

```diff
@@ -29,6 +29,7 @@ fn-help-content() {
   cat <<help_content
     certs:add <app> CRT KEY, Add an ssl endpoint to an app. Can also import from a tarball on stdin
     certs:chain CRT [CRT ...], [NOT IMPLEMENTED] Print the ordered and complete chain for the given certificate
+    certs:show <app> [key|crt], Show the server.key or server.crt on stdout
```

Comment:
Oh right, yeah this should be `<key|crt>`. My bad.

Reply:
Fixed.